### ISSUE TITLE
support remote-ssh extension and update remoting.md

### ISSUE THREAD

**Issue by chuanjiao10** [PowerShell/vscode-powershell#2036]
hi :  

debug with remote-ssh extension
------------
I use newest version of vscode and newest version of remote-ssh extension，
can open remote file and terminal(bash)，but powershell debug  do not work.

how can i edit launch.json？ 
I use '/root/.vscode-server/extensions/ms-vscode.powershell-2019.5.0/examples/.vscode/launch.json
' , every launch item not work.

debug document with remote-ssh extension
------------
https://github.com/PowerShell/vscode-powershell/blob/master/docs/remoting.md
or 
https://docs.microsoft.com/en-us/powershell/scripting/components/vscode/using-vscode-for-remote-editing-and-debugging?view=powershell-6
need update.
Should I give feedback here?

**Comment by TylerLeonhardt** [comment]
Yes so the doc does need to be updated to say "use the remote-ssh extension instead of what's in the doc today".

But we also should figure out why the remote-ssh extension is failing for you. Can you explain how you're using the remote-ssh extension?

Can you also [attach the logs](https://github.com/PowerShell/vscode-powershell/blob/master/docs/troubleshooting.md#providing-information-about-your-environment) here so we can get a better idea of what's going on?

**Comment by chuanjiao10** [comment]
1 Install the latest version of vscode.
2 install the latest version of vscode extension: remote-ssh, remote-ssh-explorer
3 Define the ssh configuration file path:
Edit c:\Users\your username\AppData\Roaming\Code\User\settings.json
Add a line:
```
"remote.SSH.configFile": "d:\\vscode\\remote.txt"
```
4 Write the contents of the ssh configuration file: d:\vscode\remote.txt, bom header +utf8 format.
Host 吃鸡ps62
  User root
  HostName 192.168.111.62
  IdentityFile ~/.ssh/id_rsa

Support multiple servers and support Chinese.

5 Download the extracted ssh-keygen key pair with https://github.com/PowerShell/Win32-OpenSSH/releases.

6 Use my ssh-copy-id6.ps1 to deploy the local public key to each server.
https://github.com/PowerShell/Win32-OpenSSH/issues/341



7 Click in the lower left corner and click 吃鸡62. Found a [new ssh sub-vscode window].
This window is actually installed in linux, linux version of vscode.
Local vscode, secretly installed remote vscode, ~/.vscode-server
You can install the Chinese extension in this [new ssh subvscode window] -> extension.
You can install the xxx language extension in this [new ssh subvscode window] -> extension.
With this extension, you can highlight and complete the code, just like the native vscode.

8 In this [new ssh sub-vscode window], install the powershell extension.

9 put '/root/.vscode-server/extensions/ms-vscode.powershell-2019.5.0/examples/.vscode/launch.json 
'， to /powershell/.vscode

10 reopen vscode . click on the folder to open, browse the directory on linux, and open  /powershell folder. open a ps1 file. make breakpoint. and  ```f5```

11-point terminal---"New terminal.
You can open the command line of [that linux]. ```bash```

12 Look at the bottom left corner and you know which linux machine you are connected to.


========================

I installed a Linux vm half a year ago. On the linux gnome desktop, I installed the linux version of vscode and powershell extension. 
It is possible to debug ps1  on locally Linux. 
I think the remote-ssh extension is similar to installing vscode on Linux. 
I don't know why I can't debug now?

**Comment by rjmholt** [comment]
One thing I see:

```
VERBOSE: Invoking Start-EditorServicesHost
/usr/bin/chmod: 无法访问"/tmp/CoreFxPipe_PSES_pcrxxkzd.vtf": 没有那个文件或目录
/usr/bin/chmod: 无法访问"/tmp/CoreFxPipe_PSES_pcrxxkzd.vtf": 没有那个文件或目录


Permissions to the pipe file were not set properly. Expected: 600 Actual:  for file: /tmp/CoreFxPipe_PSES_pcrxxkzd.vtf
```

**Comment by TylerLeonhardt** [comment]
Ah. Interesting. In the [Preview extension (which you can get here)](https://devblogs.microsoft.com/powershell/announcing-the-powershell-preview-extension-in-vscode/) we changed the pipe permission logic. @chuanjiao10 can you try the PowerShell Preview extension and see if this issue still repros?

**Comment by chuanjiao10** [comment]
@TylerLeonhardt   ok!!!  thanks！！！

1 I delete /powershell/.vscode/launch.json
2 disable  powershell extension,then install  powershell-preview extension.
3 restart,open folder /powershell.
4 ```f5```
**Breakpoint, single step, completely normal.**

**Comment by TylerLeonhardt** [comment]
Awesome! Glad it's working for you 🙂

**Comment by chuanjiao10** [comment]
Looking forward to the stable version of the powershell extension, early support for remote-ssh

**Comment by chuanjiao10** [comment]
https://user-images.githubusercontent.com/8968667/59777628-9962cb00-92e7-11e9-8f66-5460947f0f8b.png

hi all:
i just tested,
ms-vscode.powershell-2019.9 ,can not debug yet.
ms-vscode.powershell-preview-2019.9 ,debug good yet.

**Comment by TylerLeonhardt** [comment]
Yes that logic was not brought into stable (preview is more of a fork that will replace stable one day, rather than being incremental). 

Let me see if I can make that change for you. I fear that it could be a potentially risky change to move it over to stable but I'll give it a go.

**Comment by chuanjiao10** [comment]
en ok ,thanks to reply
